protected_test.go

```diff
@@ -25,7 +25,7 @@ func (p *testprotector) Protect(fileDescriptor int) error {
 
 func TestConnectIP(t *testing.T) {
 	p := &testprotector{}
-	pt := New(p.Protect, "8.8.8.8")
+	pt, _ := New(p.Protect, "8.8.8.8:53")
```

Comment:
@oxtoacart  I ended up passing dns IP instead, to bypass the IPv6 issue you encountered.